src/posts/2022-11-30-gsod/index.md

```diff
@@ -0,0 +1,121 @@
+---
+path: '/blog/2022/11/30/cilium-google-season-of-docs-2022'
+date: '2022-11-30T17:00:00.000Z'
+title: 'Cilium Google Season of Docs Case Study'
+ogImage: GSoD.png
+ogSummary: "Learn how Google Season of Docs went for Cilium"
+categories:
+  - Community
+tags:
+  - Cilium
+---
+
+**Cilium Google Season of Docs Case Study: Reorganize Cilium Features and Getting Started**
+
+*Authors: _Bill Mulligan_*
+
+*November 30th, 2022* 
+
+Organization or Project: [Cilium](https://docs.cilium.io/en/v1.12/contributing/development/season_of_docs/)
+
```

Comment:
This looks strange on our blog. The link to the original proposal seems useful but maybe better linked from somewhere in the text rather than link this?

Reply:
I like the suggestion below so I'll remove it